displayTable.py

```diff
@@ -5,9 +5,10 @@
 	@param wiki - Language code for the wiki (eg: 'en', 'es', 'it')
 	@param content - Python 2D array (nested list) containing table entries
 	@param desc - Key to the description message of the report
+	@param numbers - If table have to contain numbers of rows
```

Comment:
I still see this in the diff. Better remove it completely and we can add something to automatically add line numbers. That would be simpler.